src/zjs_ashell_ide.json

```diff
@@ -0,0 +1,42 @@
+{
+    "module": "ashell_ide",
+    "depends": ["aio", "arduino101_pins", "ble",
+                "common", "console", "fs", "gpio", "grove_lcd", "i2c",
+                "performance", "pwm", "sensor_accel", "sensor_gyro",
+                "sensor_light", "sensor_temp", "spi"],
```

Comment:
Based on my previous comment, in the _cli and _ide versions I think this can just contain "ashell" and nothing else.